05-jupyter-ecosystem.md

```diff
@@ -374,7 +374,7 @@ curriculum.
 
 ## Gotchas
 
-### Programming language ≠ Jupyter
+### Programming language $\neq$ Jupyter
```

Comment:
This issue can also be remedied by choosing a font that contains the unicode characters. For example, this is from `master` using Source Serif Pro.
![a](https://user-images.githubusercontent.com/3303/49397120-f86c4400-f6f7-11e8-9937-9cb23385b49f.png)
Does anyone have font preferences for the PDF?

Reply:
(That typeface for the print version is so much nicer than Computer Modern.)

Reply:
@labarba Let's decide on a font (this one or another) for the PDF before deciding whether to merge this PR. It's probably avoidable by choosing a font with more complete unicode coverage, which we're likely to do anyway.

Cc: @tgeorgeux @kyleniemeyer (people who I imagine have opinions on this matter). Do we need to solicit broader feedback or just make a decisions.

If you want to reproduce locally, I did
```
diff --git i/index.Rmd w/index.Rmd
index c971f87..868981c 100644
--- i/index.Rmd
+++ w/index.Rmd
@@ -24,6 +24,9 @@ documentclass: book
 bibliography: book.bib
 biblio-style: apa
 link-citations: yes
+mainfont: Source Serif Pro
+sansfont: Source Sans Pro
+codefont: Source Code Pro
 github-repo: jupyter4edu/jupyter-edu-book
 description: "A handbook on teaching and learning with Jupyter notebooks."
 ---
```

Reply:
In my opinion, having to pick a font that supports particular unicode will make this more brittle than it needs to be. I would rather:

* avoid unicode altogether (spell out "does not equal")
* use LaTeX where we know it will work

We want this to work with as many output formats as we can.